docs/quick-start/connecting-mysql/page.md

```diff
@@ -1,26 +1,32 @@
-# Connecting MySQL
+# Connecting MariaDB/MySQL
 
-Just like Redis GoFr also supports connection to SQL(MySQL and Postgres) databases based on configuration variables.
+Just like Redis GoFr also supports connection to SQL(MariaDB, MySQL and Postgres) databases based on configuration variables.
 
 ## Setup
 
-Users can run MySQL and create a database locally using the following Docker command:
+Users can run MariaDB/MySQL and create a database locally using the following Docker command:
```

Comment:
I will love if we can just reverse the order of showing docker commands to run `mariaDB` and `MySQL`, since the title of page is **connecting with mysql** we should keep it above and shift the `MariaDB` command beneath that.